lib/utils/forge-core.js

```diff
@@ -0,0 +1,14 @@
+const { electronProjectPath } = require('./build-paths');
+
+try {
+  module.exports = require(require.resolve('@electron-forge/core', {
+    paths: [ electronProjectPath ]
+  }));
+} catch (e) {
+  throw new Error([
```

Comment:
Just curious, why `new Error(...)` instead of [shorthand `Error(...)`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error)?

Reply:
I'm not sure what you're talking about, I don't see any description of a shorthand form in that link. Error is a class and I instantiate classes with new 🤷

Reply:
Description > Used as a function:

> When Error is used like a function -- without new, it will return an Error object. Therefore, a mere call to Error will produce the same output that constructing an Error object via the new keyword would.

Reply:
I just peeked at Ember.js' source and see that they use/prefer `new Error` so I guess I'm in the minority. (I like to remove code that I don't think adds meaning so always omit the `new` for `Error`) Just wondered if you had some seasoned opinion that I could learn from.

Reply:
🤷‍♂ I guess I don't like taking advantage of unique behaviors when standard behaviors apply since I think it makes code harder to read by adding mental load (have to remember that `Error`, unlike other constructors, can be invoked without `new`). Also makes it harder for people coming from other languages.

Reply:
Fair enough -- thanks for taking a minute to share your thoughts.